tests/Cookie/CookieJarTest.php

```diff
@@ -37,6 +37,19 @@ public function testCreatesFromArray()
         $this->assertCount(2, $jar);
     }
 
+    public function testGetsCookiesByName()
+    {
+        $cookies = $this->getTestCookies();
+        foreach ($this->getTestCookies() as $cookie) {
+            $this->jar->setCookie($cookie);
+        }
+
+        $testCookie = $cookies[0];
+        $this->assertEquals($testCookie, $this->jar->getCookieByName($testCookie->getName()));
+        $this->assertNull($this->jar->getCookieByName("doesnotexist"));
+        $this->assertNull($this->jar->getCookieByName(null));
```

Comment:
Done. I pushed changes that just return null when `$name` is null.
Or do you prefer to throw an exception when this happens?